### PR TITLE
update: made rules UI page

### PR DESCRIPTION
**Pull request by simran61**
Imported UI for rules page

rules

### CODE REVIEW

File: src/Pages/Rules/Rules.js [hunk 1]
```diff
@@ -1,7 +1,49 @@
 import React from "react";
+import "./Rules.css";
+import leaderboard from "../../assests/leaderBoard.png";
 
 const Rules = () => {
-  return <div>Rules</div>;
+  return (
+    <>
+      <div class="rules_box">
+        <div class="rules_content">
```
Comment (znjs) [inline src/Pages/Rules/Rules.js:9]: Change all the `class` to `className` in this file

**Review by znjs — APPROVED** [review]
- clean UI 
- Good PR

File: src/Pages/Rules/Rules.js (hunk 1 above)
Comment (simran61) [reply src/Pages/Rules/Rules.js]: Updated. Thanks :)

File: src/Pages/Rules/Rules.js [hunk 2]
```diff
@@ -1,7 +1,50 @@
...
+import { Link } from "react-router-dom";
 
 const Rules = () => {
-  return <div>Rules</div>;
+  return (
+    <>
+      <div className="rules_box">
+        <div className="rules_content">
+          <h1 className="heading_txt">Rules of the quiz!</h1>
+          <p className="para_txt">
```
Comment (Launchpad5682) [inline src/Pages/Rules/Rules.js:12]: This para element is repeating. You can extract it to a separate component.

**Review by Launchpad5682 — APPROVED** [review]

File: src/Pages/Rules/Rules.js (hunk 2 above)
Comment (simran61) [reply src/Pages/Rules/Rules.js:12]: okk will try. Thanks!

**Pull request merged**